psamm_import/importer.py

```diff
@@ -556,6 +562,12 @@ def main():
         logging.basicConfig(
             level=logging.INFO, format=u'%(levelname)s: %(message)s')
 
+    if(os.path.isfile(args.dest + '/model.yaml')and not args.force):
+        raise OverwriteError('Files will be overwritten, but if you want to'
+                             ' continue with the import add `--force` to your'
+                             ' command.')
+        return
+
```

Comment:
Also, if `--force` is set and the directory exists I would still report "Destination directory is not empty, overwriting existing files in {}" but with `logger.warning()` instead or `logger.error()`.